src/SprykerEco/Yves/Adyen/Theme/default/components/molecules/credit-card/credit-card.twig

```diff
@@ -0,0 +1,42 @@
+{% extends model('component') %}
+
+{% define config = {
+    name: 'credit-card',
+    tag: 'credit-card'
+} %}
+
+{% define data = {
+    form: required,
+    scriptSrc: data.form.adyenCreditCard.vars.sdkUrl,
+    configKey: data.form.adyenCreditCard.vars.sdkOriginKey
+} %}
+
+{% define attributes = {
+    'client-config-key': data.configKey
+} %}
+
+{% block body %}
+    {% include molecule('script-loader') with {
+        attributes: {
+            src: data.scriptSrc
+        }
+    } only %}
+    <div class="col col--sm-12 col--md-6 js-chckt-pm__pm-holder {{ config.jsName }}__form">
```

Comment:
`js-chckt-pm__pm-holder` is this class needed?

Reply:
removed